SIAB/spillage/api.py

```
@@ -113,8 +118,8 @@ def _coef_opt_jy(rcut, siab_settings, folders, options, nthreads):
     """
     minimizer = Spillage_jy()
     print(f"ORBGEN: Optimizing orbitals for rcut = {rcut} au", flush = True)
-
-    orbparams = siab_settings['orbitals']
+    if spill_coefs:
+        print(f"ORBGEN: For jy, spill_coefs is deprecated.", flush = True)
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)